src/prpl.ts

```diff
@@ -156,7 +164,15 @@ self.addEventListener('activate', () => self.registration.unregister());`);
       // We handle the caching header ourselves.
       cacheControl: false,
     };
-    send(request, fileToSend, sendOpts).pipe(response);
+
+    let stream = send(request, fileToSend, sendOpts);
+
+    // Set a custom error-handling function
+    if(config && config.error) {
```

Comment:
Can you run `npm run format` to fix up the formatting?

Reply:
It should be correct now.